src/Npgsql.NodaTime/Internal/NodaTimeTypeInfoResolver.cs

```diff
@@ -84,14 +96,73 @@ static void AddInfos(TypeInfoMappingCollection mappings)
 
         // interval
         mappings.AddType<Period>(IntervalDataTypeName,
-            static (options, mapping, _) => mapping.CreateInfo(options, new IntervalPeriodConverter()), isDefault: true);
+            static (options, mapping, _) => mapping.CreateInfo(options, new PeriodConverter()), isDefault: true);
         mappings.AddStructType<Duration>(IntervalDataTypeName,
-            static (options, mapping, _) => mapping.CreateInfo(options, new IntervalDurationConverter()));
+            static (options, mapping, _) => mapping.CreateInfo(options, new DurationConverter()));
+
+        // daterange
```

Comment:
Alright so these would move out in the future factory reality

Reply:
Yep